resources/views/home/user.blade.php

```diff
@@ -22,7 +22,7 @@
         'title' => trans('home.user.header.welcome', ['username' => link_to(
             route('users.show', Auth::user()),
             Auth::user()->username,
-            ['class' => 'link link--white link--no-underline']
```

Comment:
`.link--underline` is not a valid class, you probably want to just remove it. `.link--no-underline` is what disables the underline on hover, so removing this class will do what you want.